content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -190,25 +155,42 @@ As outlined above, the [Flannel](https://github.com/coreos/flannel) CNI [meta pl
 
 For the node, pod, and service objects, the following network flows are supported for TCP/UDP traffic:
 
-* Pod -> Pod (IP)
-* Pod -> Pod (Name)
-* Pod -> Service (Cluster IP)
-* Pod -> Service (PQDN, but only if there are no ".")
-* Pod -> Service (FQDN)
-* Pod -> External (IP)
-* Pod -> External (DNS)
-* Node -> Pod
-* Pod -> Node
+* Pod → Pod (IP)
+* Pod → Pod (Name)
+* Pod → Service (Cluster IP)
+* Pod → Service (PQDN, but only if there are no ".")
+* Pod → Service (FQDN)
+* Pod → external (IP)
+* Pod → external (DNS)
+* Node → Pod
+* Pod → Node
+
+#### CNI plugin limitations
 
-##### IP address management (IPAM) {#ipam}
+* Windows reference network plugins win-bridge and win-overlay do not implement [CNI spec](https://github.com/containernetworking/cni/blob/master/SPEC.md) v0.4.0, due to a missing "CHECK" implementation.
+* The Flannel VXLAN CNI plugin has the following limitations on Windows:
```

Comment:
@sftim, Would you double check this list, starting at line 170?
See ,https://deploy-preview-26848--kubernetes-io-master-staging.netlify.app/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#cni-plugin-limitations